virttest/remote_commander/remote_master.py

```diff
@@ -259,6 +265,15 @@ def listen_errors(self, cmd):
                             remote_interface.MessengerError)):
             raise cmd
 
+    def listen_queries(self, cmd):
+        """Manage queries from slave side.
+        """
```

Comment:
Yes, exactly why it was written for. I want to pass Python objects. So, is it okay ?)